main.go

```diff
@@ -126,14 +132,15 @@ func main() {
 		}
 
 		if healthy {
-			if silenceReq.Active() {
```

Comment:
Note to reviewers: This simplification because `silenceReq.Active()` is always true here.